lib/tasks/data_updates.rake

```diff
@@ -0,0 +1,7 @@
+namespace :data_updates do
+  desc "Enqueue Sidekiq worker to handle data updates"
+  task enqueue_data_update_worker: :environment do
+    # Ensure new code has been deployed before we run our update scripts
+    DataUpdateWorker.perform_in(10.minutes)
```

Comment:
When running locally, isn't there a good chance this won't get run since before the server is killed? Perhaps run immediately when `Rails.env == 'development'`?

```suggestion
    if Rails.env.development?
      DataUpdateWorker.perform
    else
      DataUpdateWorker.perform_in(10.minutes)
    end
```

Reply:
Good call!!! Actually, what do you think about running it inline? I would think any local dataset is going to be relatively small so it likely won't take that long and then you know once `bin/update` is complete you are ready to go, rather than having to check workers.

Reply:
Sounds great!

Reply:
+1 for inlining this in development.